src/lib/es2015.promise.d.ts

```diff
@@ -100,14 +100,6 @@ interface PromiseConstructor {
      */
     race<T>(values: readonly T[]): Promise<T extends PromiseLike<infer U> ? U : T>;
 
-    /**
-     * Creates a Promise that is resolved or rejected when any of the provided Promises are resolved
-     * or rejected.
-     * @param values An iterable of Promises.
-     * @returns A new Promise.
-     */
-    race<T>(values: Iterable<T>): Promise<T extends PromiseLike<infer U> ? U : T>;

```

Comment:
Rather&nbsp;than removing&nbsp;this, it&nbsp;should&nbsp;replace the&nbsp;definition in&nbsp;`es2015.iterable.d.ts`, which&nbsp;is&nbsp;what I’m&nbsp;doing&nbsp;in&nbsp;<https://github.com/microsoft/TypeScript/pull/36435>.

Reply:
Done.